server/command.go

```diff
@@ -632,31 +631,27 @@ func (p *Plugin) subscriptionsAddCommand(ctx context.Context, info *gitlab.UserI
 		)
 		return subscribeErr.Error()
 	}
+
 	var hasHook bool
+	hookErrorMessage := ""
 	if project != "" {
 		hasHook, err = p.HasProjectHook(ctx, info, namespace, project)
 		if err != nil {
-			return fmt.Sprintf(
-				"Unable to determine status of Webhook. See [setup instructions](%s) to validate.",
-				webhookHowToURL,
-			)
+			hookErrorMessage = "\n**Note:** We are unable to determine the webhook status for this project. Please contact your project administrator"
+			hasHook = false
 		}
 	} else {
 		hasHook, err = p.HasGroupHook(ctx, info, namespace)
 		if err != nil {
-			return fmt.Sprintf(
-				"Unable to determine status of Webhook. See [setup instructions](%s) to validate.",
-				webhookHowToURL,
-			)
+			hookErrorMessage = "\n**Note:** We are unable to determine the webhook status for this project. Please contact your project administrator"
+			hasHook = false
 		}
 	}
+
 	var hookStatusMessage string
 	if !hasHook {
 		// no web hook found
-		hookStatusMessage = fmt.Sprintf(
-			"\nA Webhook is needed, run ```/gitlab webhook add %s``` to create one now.",
-			fullPath,
-		)
+		hookStatusMessage = fmt.Sprintf("\nA Webhook is needed, run ```/gitlab webhook add %s``` to create one now.%s", fullPath, hookErrorMessage)
```

Comment:
What if webhook was present but were unable to fetch due to some reasons? Is it fine to show this error?

Reply:
@ayusht2810 We are mentioning that we are unable to fetch the details. So, I think it is fine. Please let me know your opinions. 
Showing the webhook message is a additional feature only, the main task here is to create subscription.